pkg/registry/client.go

```diff
@@ -473,7 +473,6 @@ func (c *Client) Pull(ref string, options ...PullOption) (*PullResult, error) {
 		return nil, err
 	}
 
-	descriptors = append(descriptors, manifest)
```

Comment:
We should not be removing this?

Reply:
The manifest descriptor exists within 'layers' to my knowledge. I can re-add this if that layer descriptor duplication is intentional